src/bin/bat/clap_app.rs

```diff
@@ -402,6 +402,20 @@ pub fn build_app(interactive_output: bool) -> Command {
                 .long_help("Set the maximum number of consecutive empty lines to be printed.")
                 .hide_short_help(true)
         )
+        .arg(
+            Arg::new("strip-ansi")
+                .long("strip-ansi")
+                .overrides_with("strip-ansi")
+                .value_name("when")
+                .value_parser(["auto", "always", "never"])
+                .default_value("never")
+                .hide_default_value(true)
+                .help("Strip colors from the input (auto, always, *never*)")
+                .long_help("Specify when to strip ANSI escape sequences from the input. \
+                The automatic mode will remove escape sequences unless the detect language \
```

Comment:
```suggestion
                The automatic mode will remove escape sequences unless the syntax highlighting language \
```
"unless the detect language" doesn't quite read right to me. "detected" could work but as it can be explicitly set, I feel calling it "the language used for syntax highlighting" or similar flows better despite being more verbose

Reply:
Oops, good catch. That was supposed to be "detected," but appears to have lost the "-ed" somewhere along my revisions to it.